README.md

```diff
@@ -94,6 +94,16 @@ the COOL environment.
 
 | Name | Source | Version |
 |------|--------|---------|
+| cw\_alarms\_assessor\_portal | github.com/cisagov/instance-cw-alarms-tf-module | n/a |
+| cw\_alarms\_debiandesktop | github.com/cisagov/instance-cw-alarms-tf-module | n/a |
+| cw\_alarms\_gophish | github.com/cisagov/instance-cw-alarms-tf-module | n/a |
+| cw\_alarms\_guacamole | github.com/cisagov/instance-cw-alarms-tf-module | n/a |
+| cw\_alarms\_kali | github.com/cisagov/instance-cw-alarms-tf-module | n/a |
+| cw\_alarms\_nessus | github.com/cisagov/instance-cw-alarms-tf-module | n/a |
+| cw\_alarms\_pentestportal | github.com/cisagov/instance-cw-alarms-tf-module | n/a |
+| cw\_alarms\_samba | github.com/cisagov/instance-cw-alarms-tf-module | n/a |
+| cw\_alarms\_teamserver | github.com/cisagov/instance-cw-alarms-tf-module | n/a |
+| cw\_alarms\_terraformer | github.com/cisagov/instance-cw-alarms-tf-module | n/a |
```

Comment:
Would you re-generate from `terraform-docs` to pick up the `cw_alarms_windows` module?